### PR TITLE
Create Support

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2245](https://jira.bigcommerce.com/browse/DEVDOCS-2245)

## What changed?

- Created a Support article.
- Used the following outline, added a blurb and link for each item
*Dev Community
*Developer Blog
*Dev Library
*Dev Slack 
*Changelog
*Town Hall

### CODE REVIEW

File: docs/api-docs/partner/Support [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Support
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Developer community](#Dev-community)
```
Comment (ektravel) [inline docs/api-docs/partner/Support:6]:
```suggestion
- [Developer community](#developer-community)
```

File: docs/api-docs/partner/Support [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+- [Developer blog](#Dev-blog)
```
Comment (ektravel) [inline docs/api-docs/partner/Support:7]:
```suggestion
- [Developer blog](#developer-blog)
```

File: docs/api-docs/partner/Support [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+- [Developer library](#Dev-library)
```
Comment (ektravel) [inline docs/api-docs/partner/Support:8]:
```suggestion
- [Developer library](#developer-library)
```

File: docs/api-docs/partner/Support [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+- [Dev Slack](#Dev-slack)
```
Comment (ektravel) [inline docs/api-docs/partner/Support:9]:
```suggestion
- [Dev Slack](#dev-slack)
```

File: docs/api-docs/partner/Support [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+- [Changelog](#Changelog)
```
Comment (ektravel) [inline docs/api-docs/partner/Support:10]:
```suggestion
- [Changelog](#changelog)
```

File: docs/api-docs/partner/Support [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+- [Town Hall](#Town-Hall)
```
Comment (ektravel) [inline docs/api-docs/partner/Support:11]:
```suggestion
- [Town Hall](#town-hall)
```

File: docs/api-docs/partner/Support [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+</div> 
+
+## Developer community
```
Comment (ektravel) [inline docs/api-docs/partner/Support:14]:
```suggestion
## Developer community group
```

File: docs/api-docs/partner/Support [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+The [Developer community](https://support.bigcommerce.com/s/group/0F913000000HLjECAW/bigcommerce-developers) is a forum for web developers working with BigCommerce APIs, creating a custom theme, or building a custom app or integration. Find answers, browse topics, and talk to other developers in this group.
+
+## Developer blog
+The [Developer blog](https://developer.bigcommerce.com/) provides product, industry, how-to, and community information regularly. Find news, tips, and stories for developing on BigCommerce.
```
Comment (ektravel) [inline docs/api-docs/partner/Support:18]:
```suggestion
The [Developer blog](https://medium.com/bigcommerce-developer-blog) provides product, industry, how-to, and community information regularly. Find news, tips, and stories for developing on BigCommerce. 
```

File: docs/api-docs/partner/Support [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+</div> 
+
+## Developer community
+The [Developer community](https://support.bigcommerce.com/s/group/0F913000000HLjECAW/bigcommerce-developers) is a forum for web developers working with BigCommerce APIs, creating a custom theme, or building a custom app or integration. Find answers, browse topics, and talk to other developers in this group.
+
+## Developer blog
+The [Developer blog](https://developer.bigcommerce.com/) provides product, industry, how-to, and community information regularly. Find news, tips, and stories for developing on BigCommerce. 
+
+## Developer library
+The [Dev library](https://developer.bigcommerce.com/dev-library/) provides samples, tools, and guides to help jump-start the perfect ecommerce experience. Find examples and starters to help you to begin working on BigCommerce.
```
Comment (ektravel) [inline docs/api-docs/partner/Support:21]:
```suggestion
The [Developer library](https://developer.bigcommerce.com/dev-library/) provides samples, tools, and guides to help jump-start the perfect ecommerce experience. Find examples and starters to help you begin working on BigCommerce.
```

File: docs/api-docs/partner/Support [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+## Developer Slack
+[Slack developer tools](https://api.slack.com/) contain tools, libraries, and SDKs to help you start developing on slack quickly.
```
Comment (ektravel) [inline docs/api-docs/partner/Support:24]:
```suggestion
BigCommerce has a dedicated Slack channel for Tech and Agency partners. It is a great place to connect, learn about new updates, and attend “Ask Me Anything” sessions with members of the BigCommerce Product Team. To join the BigCommerce developer Slack community, fill our this [form](https://docs.google.com/forms/d/e/1FAIpQLSeQApTpd4T9lEX7hPgUjTyoq0vb8UbiCiWseuurJjdMHGBaGQ/viewform). 
```

File: docs/api-docs/partner/Support [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+## Changelog
+A [Changelog](https://developer.bigcommerce.com/changelog) is a log of notable changes made to a project. Find significant changes such as new features, latest releases, and bug fixes in chronological order.
```
Comment (ektravel) [inline docs/api-docs/partner/Support:27]:
```suggestion
[Changelog](https://developer.bigcommerce.com/changelog) is a log of notable updates made to the BigCommerce platform. Find significant changes such as new features, latest releases, and bug fixes in chronological order.
```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**